go.mod

```diff
@@ -90,7 +108,7 @@ require (
 	github.com/docker/go-units v0.5.0 // indirect
 	github.com/dustin/go-humanize v1.0.1 // indirect
 	github.com/elastic/gosigar v0.14.3 // indirect
-	github.com/ethereum/go-ethereum v1.14.11 // indirect
+	github.com/ethereum/go-ethereum v1.14.11
```

Comment:
Sorry, very silly nit, but can you move this into the direct imports since it's apparently directly used by deposits (I think) now.